event_routing_backends/processors/xapi/event_transformers/navigation_events.py

```diff
@@ -166,8 +141,7 @@ def get_context(self):
                 constants.XAPI_CONTEXT_ENDING_POSITION: 'previous unit',
             })
 
-        context = Context(
-            contextActivities=self.get_context_activities()
-        )
+        context = super().get_context()
```

Comment:
I think we should rename the method to `get_context_extensions` and return extensions from this method